src/lib/common/include/sol-platform.h

```diff
@@ -74,7 +74,7 @@ extern "C" {
  * When this time is reached, stops the * chunk reading and allow mainloop to run again.
  * This keeps interactivity.
  */
-#define CHUNK_MAX_TIME_NS (20 * (NSEC_PER_MSEC))
+#define CHUNK_MAX_TIME_NS (20 * (1000000000ULL))
```

Comment:
i'll see it, but just to record, it is already happening, this value is defined in `sol-util.h` and there is any check there.